src/api/websocket.ts

```diff
@@ -0,0 +1,22 @@
+// TODO: This is just a dumb client to test the websocket connection
+// TODO: This needs to be connected to the redux store
```

Comment:
We can experiment with putting this into a React Component which would handle websocket errors, reconnecting and dispatching the actions based on notification type